### PR TITLE
Corrected Polish locale for 2.0.

### PR DESCRIPTION
**Pull request by jarosz**
General names.
Fix TODOs - in Polish primary patron is called 'główny' and secondary has no extra word.
Lowercase bishop title.
Re-translate and lowercase priest title.
Lowercase martyr title.
Commas for companions.
Commas for patrons.
Some other typos.

Add missing translations for keys from calendars/poland.js:
* joseph_sebastian_pelczar_bishop
* our_lady_queen_of_poland
* our_lady_help_of_christians
* hedwig_of_poland
* dedication_of_consecrated_churches

### CODE REVIEW

**Comment by emagnier** [comment]
Thanks for your quick contribution! Let's wait for @tukusejssirs comments to then merge this PR asap.

File: src/locales/pl.ts
Comment (tukusejssirs) [inline src/locales/pl.ts:93]: As we try to use the celebration names from the official translation of Roman Missal in the particular vernacular, I think this should be changed to [`Trójcy Przenajświętszej`](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf#page=83).

Also note that I haven’d proofed any other celebration names. @jarosz, could you please proof the celebration names from the Polish Roman Missal? Ideally from _editio typica tertia_, however, if you can’t access it (or it is not published yet in Polish), you could use _editio typica altera_ or even the above-linked [1986 _editio  typica_](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf).

File: src/locales/pl.ts
Comment (tukusejssirs) [inline src/locales/pl.ts:112]: @jarosz, could you add another key for this celebration? The original key name (without the `patrona Polski`) is also used in Lithuania and Ukraine.

Also, try to localise it other other locales where the ‘base’ celebration (the one without the patronage title) is already present (it should be just a simple copy-paste work).

File: src/locales/pl.ts
Comment (tukusejssirs) [inline src/locales/pl.ts:182]: Could you remove the trailing whitespace please? Even better: could you install and use [`.editorconfig`](editorconfig.org) please? :smiley:

File: src/locales/pl.ts
Comment (tukusejssirs) [inline src/locales/pl.ts:299]: I am not sure of the more recent editions, however, in the [_editio typica_](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf#page=850), they don’t use a comma before `i Towarzyszy`.

**Review by tukusejssirs — CHANGES_REQUESTED** [review]
I’ve finished my ‘quick’ review of this PR. I haven’t gone any deeper (like comparing all the key values with the official Polish translation of Roman Missal); I just simply read all the changes.

Note that I am not sure that all my comments are 100 % correct; do check upon it please.

File: src/locales/pl.ts
Comment (tukusejssirs) [inline src/locales/pl.ts:400]: @jarosz, although it is not yet in the `_F-TypeScript` nor `dev` branch, `wenceslaus_i_of_bohemia_martyr_patron_of_czech_nation` will be added as soon as #248 will be merged. Please, review my Polish translation of the key below (more info in [this comment](https://github.com/romcal/romcal/pull/248#discussion_r518015314)). Thanks!

```json
pl : 'Św. Wacława, męczennika i patrona narodu czeskiego',
```

**Review by tukusejssirs — COMMENTED** [review]
@jarosz, I have added a review comment long time ago [some day(s) after you created this PR), but I forgot to submit the review and thus my comment did not show up.

The comment is not a blocker, just an additional note. :smiley:

@emagnier, this PR is ready to be merged from my point of view. :smiley:

**Comment by emagnier** [comment]
@tukusejssirs: as you are suggesting, we could merge this PR, since there is already quite a lot of improvements here. The remaining notes could be addressed later. Are you still ok with that?

**Comment by tukusejssirs** [comment]
> @tukusejssirs: as you are suggesting, we could merge this PR, since there is already quite a lot of improvements here. The remaining notes could be addressed later. Are you still ok with that?

Indeed. I’ll find some time to create an issue from these comments in an hour or so.

**Pull request merged**
